extensions/data-workspace/package.nls.json

```diff
@@ -6,7 +6,7 @@
 	"new-project-command": "New Project",
 	"refresh-workspace-command": "Refresh",
 	"remove-project-command": "Remove Project",
-	"projects-view-no-workspace-content": "A workspace is required in order to use the project feature.\n[Open Workspace…](command:workbench.action.openWorkspace)\n[Save Workspace…](command:workbench.action.saveWorkspaceAs)\n",
-	"projects-view-no-project-content": "No projects found in current workspace.\n[New Project…](command:projects.newProject)\n[Open Project…](command:projects.openProject)\n",
+	"projects-view-no-workspace-content": "[Create new](command:projects.newProject)\n[Open existing](command:projects.openProject)\n",
```

Comment:
hmm, I still prefer to keep the button text, in the future when we introduce data sources to this view container, 'Create New' and Open Existing will be confusing.

Reply:
The change from "Open Project" to "Open Existing" is because we're adding the option to open a workspace in the open dialog, so you could open a project or a workspace. "Create New" could still make sense for data sources and that could be added to the dialog later on.